frontend/src/plugins/__tests__/auth.spec.js

```diff
@@ -31,6 +31,8 @@ const validJWTPayload =
 //   "user": "/users/1a2b3c4d"
 // }
 
+const envBackup = JSON.parse(JSON.stringify(window.environment))
```

Comment:
Consider creating a function `deepClone` or similar which explains why you are using `JSON.parse(JSON.stringify())` in this file, or use the lodash `clone` function instead. I was confused when first reading this.

Reply:
fixed in https://github.com/ecamp/ecamp3/pull/3222/commits/775f63269c2a2670567c089b5976f2b89f301310